src/components/PullRequestItem.tsx

```diff
@@ -186,10 +205,25 @@ export const PullRequestItem = observer((props: PullRequestItemProps) => {
           )}
         </Title>
         <PullRequestStatus pullRequest={props.pullRequest} />
-        <Repo>
-          {props.pullRequest.repoOwner}/{props.pullRequest.repoName} (#
-          {props.pullRequest.pullRequestNumber})
-        </Repo>
+        <ContextSummary>
+          <Repo>
+            {props.pullRequest.repoOwner}/{props.pullRequest.repoName} (#
+            {props.pullRequest.pullRequestNumber})
+          </Repo>
+          {props.pullRequest.changeSummary && (
+            <ChangeSummary>
```

Comment:
What do you think of adding a `title` attribute here, so people who hover can easily tell what the abbreviated form means?

Suggestion:
```suggestion
            <ChangeSummary title={`${props.pullRequest.changeSummary.additions} line(s) added, ${props.pullRequest.changeSummary.deletions} line(s) removed, ${props.pullRequest.changeSummary.changedFiles} files changed`}>
```

Reply:
> What do you think of adding a `title` attribute here, so people who hover can easily tell what the abbreviated form means?
> 
> Suggestion:

That makes a lot of sense, I'll add that. I might look into choosing the right tense, singular/plural, depending on the value but only if it's not too complex.

Reply:
Sounds good. At some stage it might even make sense to make PR Monitor localisable, but I can't say that's at the top of the priority list :)

Reply:
Have added the title. I had to repeat `props.pullRequest.changeSummary.additions` and friends a couple of times each which became quite verbose so I introduced some local variables - I don't know if doing it this way is in the preferred style. Let me know if you'd like me to change it.

<img width="600" alt="Screenshot 2021-06-03 at 10 24 11" src="https://user-images.githubusercontent.com/80959191/120613245-565b7d00-c456-11eb-8c76-d12a8baf2fff.png">